jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/kubernetes/DockerBuildService.java

```diff
@@ -70,7 +70,10 @@ protected void pushSingleImage(ImageConfiguration imageConfiguration, int retrie
         try {
             dockerServices.getRegistryService().pushImage(imageConfiguration, retries, registryConfig, skipTag);
         } catch (IOException ex) {
-            throw new JKubeServiceException("Error while trying to push the image: " + ex.getMessage(), ex);
+            String message = "Error while trying to push the image: " + ex.getMessage() +
+                             "\nPossible issue: wrong image name or registry." +
+                             "\nHint: Rename image name or registry with the jkube.generator.name property = registry name and user name and image name";
+            throw new JKubeServiceException(message, ex);
```

Comment:
I don't think we should always log this warning. IOException could also mean problems with network. It would be better if we could rely on docker response codes. If not you can check exception message for keywords like `unauthorized`/`denied` etc.

Reply:
We also need to do the same for JibBuildService as well.